MapboxNavigation/NavigationMapView.swift

```diff
@@ -1025,39 +1027,122 @@ extension Dictionary where Key == Int, Value: MGLStyleValue<NSNumber> {
 }
 
 //MARK: NavigationMapViewDelegate
```

Comment:
This `MARK` is a bit redundant to the actual declaration below. jazzy uses `MARK` comments to insert task-oriented section headers in a class reference.